badgesmodel/model.go

```diff
@@ -29,21 +29,21 @@ type Badge struct {
 }
 
 type UserBadge struct {
-	Badge
+	*Badge
```

Comment:
Why is neede to convert to pointers the embedding here?

Reply:
To avoid having to make an indirection in places like this one:
https://github.com/larkox/mattermost-plugin-badges/blob/abdca49ce6055cb6c22bbe90f3d17b623bbd5ffc/server/store.go#L212